### ISSUE TITLE
aws_budgets_budget import errors

### ISSUE THREAD

**Issue by tqggtgdl** [terraform-providers/terraform-provider-aws#6180]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Terraform Version

```bash                                                                                                                                              
Terraform v0.11.7
+ provider.aws v1.40.0
```

### Affected Resource(s)

* aws_budgets_budget

### Terraform Configuration Files

```bash
terraform import aws_budgets_budget.myBudget "123456789012:My Budget"
```

### Expected Behavior

Terraform should have imported my budget.

### Actual Behavior

```
aws_budgets_budget.myBudget: Importing from ID "123456789012:My Budget"...

Error: aws_budgets_budget.myBudget (import id: 123456789012:My Budget): import aws_budgets_budget.myBudget (id: 123456789012:My Budget): resource aws_budgets_budget doesn't support import
```

### Steps to Reproduce

1. Attempt to import an existing budget.

### References

The manual says you can use import on budgets. <https://www.terraform.io/docs/providers/aws/r/budgets_budget.html>

**Comment by bflad** [comment]
Hi @tqggtgdl 👋 Thanks for reporting this and sorry for the hassle! Turns out that while the import functionality was documented in the resource documentation, it was never actually implemented. I have submitted #6226 to actually implement it.

**Comment by bflad** [comment]
Support for resource import has been merged and will release with version 1.42.0 of the AWS provider, likely today or tomorrow. 👍

**Comment by bflad** [comment]
This has been released in [version 1.42.0 of the AWS provider](https://github.com/terraform-providers/terraform-provider-aws/blob/master/CHANGELOG.md#1420-october-31-2018). Please see the [Terraform documentation on provider versioning](https://www.terraform.io/docs/configuration/providers.html#provider-versions) or reach out if you need any assistance upgrading.
